docs/api_specification.md

```diff
@@ -974,6 +974,54 @@ Content-Type: application/json
 
 -----------------------------------------------------------------------------------------------------------
 
+## POST /v1/api/services/`{serviceExternalId}`/govuk-pay-agreement
+
+This endpoint records that a GOV.UK Pay agreement has been accepted for the service.
+
+### Request example
+```
+POST v1/api/services/7d19aff33f8948deb97ed16b2912dcd3/govuk-pay-agreement
+Content-Type: application/json
+{
+    "user_external_id": "abcde1234"
+}
+```
+
+### Request body description
+
+| Field                    | required | Description                                                    | Supported Values     |
+| ------------------------ |:--------:| -------------------------------------------------------------- |----------------------|
+| `user_external_id`       |   X      | external id of the user's whom accepted the online terms       |                      |
+
+### Response Example
+
+```
+200 OK
+```
+
+-----------------------------------------------------------------------------------------------------------
+
+### GET /v1/api/services/`{serviceExternalId}`/govuk-pay-agreement
+
+This endpoint retrieves email address of the user's and timestamp that the GOV.UK Pay terms were accepted on for the service.
```

Comment:
Minor point but perhaps slightly better worded as `This endpoint retrieves the email address of the user and timestamp...` or `This endpoint retrieves the user's email address and timestamp...`